client/src/components/MapViewList.jsx

```diff
@@ -1,56 +1,30 @@
 // @flow
 
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
 import { List } from 'antd';
 
 import '../css/MapViewList.css';
-import { getResourcesByCategory } from '../utils/api';
 import MapViewEntry from './MapViewEntry';
 
-const MapViewList = () => {
-  const [resources, setResources] = useState<Array<Resource>>([]);
-  const [selectedID, setSelectedID] = useState('');
-  const location = '505 E Healey St, Champaign, IL 61820';
-
-  useEffect(() => {
-    const fetchResources = async () => {
-      const res = await getResourcesByCategory(
-        null,
-        null,
-        null,
-        null,
-        null,
-        null,
-        null,
-        null,
-        location,
-      );
-      const newResources = [];
-      if (res != null) {
-        res.result.totalData.forEach((r) => {
-          newResources.push({
-            name: r.name,
-            city: r.city,
-            address: r.address,
-            cost: r.cost,
-            id: r._id.toString(),
-            languages: r.availableLanguages,
-            distance: r.calculatedDistance,
-          });
-        });
-      }
-      setResources(newResources);
-    };
+type Props = {
+  resources: Array<Resource>,
+};
 
-    fetchResources();
-  }, []);
+const MapViewList = (props: Props) => {
+  const { resources, setSelectedResource } = props;
+  const [selectedID, setSelectedID] = useState('');
```

Comment:
I think here, since we now have a notion of a `selectedResource`, we can probably pass that down into `MapViewList`, get rid of the `selectedID` state variable, and instead have `selected={resource.id === selectedResource.id` to be a bit cleaner :)